automatic/displayfusion/update.ps1

```diff
@@ -0,0 +1,37 @@
+﻿Import-Module AU
+
+function global:au_BeforeUpdate {
+    $Latest.ChecksumType32 = 'sha256'
+
+    Get-RemoteFiles -Purge
```

Comment:
why using Get-RemoteFiles here? You're not embedding the package.
You should either use `Get-RemoteChecksum` function, or change the call at the bottom of the script to `update -ChecksumFor 32`